### PR TITLE
feat: add rhosak UI to prod beta

### PR DESCRIPTION
**Pull request by riccardo-forina**
We want to test the new UI in production but in the beta environment, before pulling the plug on the stable one. This adds the required config.

### CODE REVIEW

File: chrome/fed-modules.json [hunk 1]
```diff
@@ -440,9 +440,34 @@
...
+        "modules": [
+            {
+                "id": "rhosak",
+                "module": "./RootApp",
+                "routes": ["/application-services/streams"]
+            }
+        ]
+    },
+    "srs": {
+        "manifestLocation": "/beta/apps/srs-ui-build/fed-mods.json",
```
Comment (karelhala) [inline chrome/fed-modules.json]: No need to include the `/beta`

```suggestion
        "manifestLocation": "/apps/srs-ui-build/fed-mods.json",
```

File: chrome/fed-modules.json [hunk 2]
```diff
@@ -440,9 +440,34 @@
...
+        "modules": []
+    },
+    "ar": {
+        "manifestLocation": "/beta/apps/sr-ui-build/fed-mods.json",
```
Comment (karelhala) [inline chrome/fed-modules.json]:
```suggestion
        "manifestLocation": "/apps/sr-ui-build/fed-mods.json",
```

**Review by karelhala — COMMENTED** [review]
Looking good, just two extra `/beta` strings in manifest locations.

File: chrome/fed-modules.json (hunk 1 above)
Comment (riccardo-forina) [reply chrome/fed-modules.json]: Thank you, done!

File: chrome/fed-modules.json [hunk 3]
```diff
@@ -440,9 +440,34 @@
...
+                "routes": ["/application-services/streams"]
+            }
+        ]
+    },
+    "srs": {
+        "manifestLocation": "/apps/srs-ui-build/fed-mods.json",
+        "modules": []
+    },
+    "ar": {
+        "manifestLocation": "/apps/sr-ui-build/fed-mods.json",
```
Comment (karelhala) [inline chrome/fed-modules.json:461]: Just a question, the key is `ar`, but it has `sr-ui-build`. Isn't it a missclick?

Comment (riccardo-forina) [reply chrome/fed-modules.json:461]: That's just a trick to expose some federated modules that are not consoledot specific federated modules in a way that allows us to use the `ScalprumComponent` component to load them without too much pain. It's likely that all those will be removed soon, we can consider them temporary. Thus the id isn't really that important.

**Comment by riccardo-forina** [comment]
Any chance we can get this merged today?

**Review by karelhala — APPROVED** [review]

**Pull request merged**